src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchNotificationSettings.java

```diff
@@ -0,0 +1,48 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.utils.LorenzColor;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorText;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneswitchNotificationSettings {
+
+    @Expose
+    @ConfigOption(name = "Notification Text", desc = "The Text to be displayed as the Notification.")
```

Comment:
```suggestion
    @ConfigOption(name = "Notification Text", desc = "The text to be displayed as the notification.")
```
I am just not going to comment on the other spots where "Do not capitalize words that aren't the first in a sentence, proper nouns, or proper adjectives." applies.